### PR TITLE
First pass - introduction

### PR DESCRIPTION
**Pull request by TanyaStere42**

### CODE REVIEW

File: docs/researchers/codex/introduction.md [hunk 1]
```diff
@@ -5,8 +5,19 @@ title: An Introduction to Codex
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-This topic provides an overview of Codex.
+Codex is a web-based, multi-user software platform designed to help researchers collaboratively track individual animals in wildlife populations and estimate population size. Each Codex installation can be used by multiple researchers and organizations to study multiple species across disperse or narrow regions. Codex is also valuable as a catalog reconciliation tool.
```
Comment (Atticus29) [inline docs/researchers/codex/introduction.md]: disparate instead of disperse?
Or diverse?

File: docs/researchers/codex/introduction.md [hunk 2]
```diff
@@ -5,8 +5,19 @@ title: An Introduction to Codex
...
\ No newline at end of file
+This introduction covers the most common Codex interactions. It is intended to support Codex users and researchers collecting data in the field and to illustrate how to use Codex for data storage, curation, searching, and analysis.
+
+Included Wildbook features:
+
+* A browser-based user interface (UI)
+* A high performance PostgreSQL database for storing multiple wildlife-related data types
+* Two servers:
+  * Data Management Server - Coordinates UI browser display and data storage
+  * Wildbook Image Analysis ("WBIA") - Manages the computer vision pipeline and and artificial intelligence (AI) models and training tools. WBIA does not come pre-configured to detect and/or identify animals from a species. Pre-trained machine learning (ML) models for the species must be used, or new models must be created and configured using training data.
```
Comment (Atticus29) [inline docs/researchers/codex/introduction.md:20]: I'm not clear whether this should be WBIA or SAGE. It feels weird to have the word "Wildbook" in one of Codex's features.

**Review by Atticus29 — COMMENTED** [review]
I think "disperse" is an actual error.
The other I leave to your discretion.

File: docs/researchers/codex/introduction.md (hunk 2 above)
Comment (TanyaStere42) [reply docs/researchers/codex/introduction.md:20]: WBIA until we get the branding done on that platform

File: docs/researchers/codex/introduction.md (hunk 1 above)
Comment (TanyaStere42) [reply docs/researchers/codex/introduction.md]: went with diverse, although I think I did it wrong when I committed directly to the branch??

Comment (Atticus29) [reply docs/researchers/codex/introduction.md]: I think that was what you were supposed to do!

**Review by Atticus29 — APPROVED** [review]
LGTM!

**Pull request merged**